frontend/package.json

```diff
@@ -22,6 +22,7 @@
     "@types/node": "18.11.19",
     "@types/react": "18.0.27",
     "@types/react-dom": "18.0.10",
+    "autosuggest-highlight": "^3.3.4",
```

Comment:
what are you using this package for?

Reply:
For the new location picker, I copy-pasted this snippet of code from the [MUI documentation](https://mui.com/material-ui/react-autocomplete/#google-maps-place). `autosuggest-highlight` was a required dependency and it seems like it's needed to have highlighted text in the autosuggest menu.